src/tools/doubleTapZoomTool.js

```diff
@@ -0,0 +1,18 @@
+/* eslint class-methods-use-this: 0 */ // --> OFF
+import external from '../externalModules.js';
+import baseTool from '../base/baseTool.js';
+
+export default class extends baseTool {
+  constructor (name = 'doubleTapZoom') {
+    super({
+      name,
+      supportedInteractionTypes: ['touch']
+    });
+  }
+
+  doubleTapCallback (evt) {
+    const eventData = evt.detail;
+
+    external.cornerstone.fitToWindow(eventData.element);
```

Comment:
If I zoom out, then activate this tool, double tapping does fit the image to the window :+1:

It confused me at first, because the name implies that `doubleTap` zooms in? At least in my head. But the code does fire on double tap 👍

Reply:
Yeah that's weird, we should probably rename it.

Reply:
You guys have any idea of a new name? 
I agreed of changing it, maybe even 'Fit to window" tool.

Reply:
doubleTapFitToWindow seems fine